_posts/2018-11-12-cloud-cloud-computing.md

```diff
@@ -0,0 +1,89 @@
+
+Cloud services are delivered in a number of ways; Infrastructure as a service(IaaS), Platform as a service(PaaS) and Software as a service(SaaS). Here is a picture of resources you would traditionally manage.
+
+![Traditional IT model](/assets/images/blog/cloud-computing/traditional-IT.png){:class="img-responsive center"}
+
+### IaaS
+
+IaaS stands for Infrastructure as a service. With this service model, what you get, in simple terms, is a data center. Some aspects of computing are managed for  you by your service provider.
+
+Here's a picture.
+![IaaS model](/assets/images/blog/cloud-computing/Iaas.png){:class="img-responsive center"}
+
+
+
+
+From the image, you can see that  networking, storage, servers, virtualization and part of the Operating system are provided as a service.
+Examples Iaas providers include; [Digital Ocean](https://www.digitalocean.com/),[linode](https://www.linode.com/), [AWS](https://aws.amazon.com/),[vultr](https://www.vultr.com/).
+
+Say you want a Linux based server. You will go to any of the above sites. Provision yourself a virtual machine and that's it. You can now deploy whatever application you want to the machine/server, in the cloud. You can make your application accessible to anyone over the internet.
+Virtualization software, storage for your application and related libraries, operating system and networking are all provided, in the cloud. 
+
+
+You will, however, have to manage your own operating system, provide middleware example an antivirus, provide your application with data and management system for it, if your application needs a database you would have to create it, provide runtime libraries and the actual application.
+
+
+
+### PaaS
+
+Paas stands for Platform as a Service. The service provider provides you with a pre-configured infrastructure.PaaS includes middleware, runtimes, development, testing and deployment tools.
+Here's a picture;
+
+![PaaS model](/assets/images/blog/cloud-computing/paas.png){:class="img-responsive center"}
+
+
+
+All you provide is code for your application and the data.
+Example of Paas providers are [Heroku](https://www.heroku.com/), [Go daddy](https://uk.godaddy.com), etc
+
+In this case, all you will have to do is create code for your application, then upload the files for your provider to deploy.
+
+
+### SaaS
+Software as a service. The service provider fully manages the whole stack. It is an application hosted in a public domain therefore accessible to everyone.
+Here's a picture;
+
+![SaaS model](/assets/images/blog/cloud-computing/saas.png){:class="img-responsive center"}
+
+The only thing you manage is your own space. Your own environment where you have your own data.
+
+A good example of a Software as a service is Gmail provided by Google.
```

Comment:
Please list and link to other examples. Even our own product MPAYER isa SaaS. Look for a list over the internet of Popular SaaS products and mention that zegetech has great speciality in building SaaS products and we understand this very clearly.